gc/include/omrmm.hdf

```diff
@@ -227,6 +227,7 @@ typedef uintptr_t (*condYieldFromGCFunctionPtr) (OMR_VMThread *omrVMThread, uint
 		<data type="uintptr_t" name="regionSize" description="heap region size" />
 		<data type="uintptr_t" name="regionCount" description="total heap region count" />
 		<data type="uintptr_t" name="arrayletLeafSize" description="arraylet leaf size" />
+		<data type="uint8_t" name="isArrayletDoubleMappingEnabled" description="set to true if arraylet double mapping status is enabled, false otherwise" />
```

Comment:
I quick look at other boolean type of attributes in this file suggest to use uintptr_t. I know it's excessive, but at least let's be consistent.